Automattic/jetpack
# AI Assistant: add upgrade events on my-jetpack and editor nudges

Upgrading over 1000 requests is still handled differently with no events attached

## Proposed changes:
This PR adds `jetpack_ai_upgrade_contact_us` event on my-jetpack package and the editor upgrade nudges.

### Other information:


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
The event can be seen on the events page (after 10 mins or so) or using the localStorage debug setting.

Set up a site to be on 1000 requests tier and set the requests count to 1000 (or over it).
Visit the editor and see the upgrade nudge on the block or the sidebar (settings or Jetpack sidebar). Click "Contact Us", your default email app should start and the event should be triggered.

With the same setup, visit My Jetpack page, click "Upgrade" on the Jetpack AI card, you should land in the interstitial with a "Contact Us" button. Clicking it should fire your default email app and the event should be triggered.
